.github/workflows/ci.yml

```diff
@@ -53,6 +53,14 @@ jobs:
           name: test-result
           path: /home/runner/work/**/build/reports
           retention-days: 2
+      - name: Publish Test Report
+        uses: mikepenz/action-junit-report@v3.7.6
+        if: success() || failure()
```

Comment:
if success or failure, is there another option or is this everything?

( success() == !failure()) ?

Reply:
@Brobrechts I had the same question. Turns the answer is: yes, there is something else: `canceled()`
There is even `always()`, but in their own docs they recommend using `sucess() || failure()` instead: https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions